gizmo.py

```diff
@@ -0,0 +1 @@
+import gizmo
```

Comment:
No need to `import gizmo`. That is what the Github testing robot is going to do. For now, just focus on adding an `hello` function to `gizmo.py`.